var/spack/repos/builtin/packages/libpng/package.py

```diff
@@ -10,18 +10,18 @@ class Libpng(AutotoolsPackage):
     """libpng is the official PNG reference library."""
 
     homepage = "http://www.libpng.org/pub/png/libpng.html"
-    url = "https://github.com/glennrp/libpng/archive/v1.6.37.tar.gz"
+    url = "http://prdownloads.sourceforge.net/libpng/libpng-1.6.37.tar.gz"
```

Comment:
WWhy `http` instead of `https`?

Reply:
Can you also explain what would be the advantage of this PR compared to the current state?

Reply:
Thanks for reviewing. https was an oversight, I have updated the scheme in the commit. The benefit is that we are getting the archive from the official download URL. Also, Sourceforge (via their RSS feed) has file checksums which can be used to verify the file - github does not provide the checksums for file verification.